react/__snapshots__/examples.spec.jsx.snap

```diff
@@ -3307,6 +3307,12 @@ exports[`Label should render examples: Label 3`] = `
 </div>"
 `;
 
+exports[`Labs/ExperimentalModal should render examples: Labs/ExperimentalModal 1`] = `
+"<div>
+  <div><button>Toggle modal</button></div>
```

Comment:
Can the modal be opened for snapshots ?

Reply:
We need to figure out why all our Path/Components are empty div. 

See https://github.com/cozy/cozy-ui/pull/1039

Will take a look tomorrow